### DISCUSSION TITLE
Endorsement

### DISCUSSION THREAD

**Discussion by mcardia** [jhuckaby/Cronicle#443] [General]
Hi!

I'm looking for a production ready scheduler. 
Does Cronicle has been tested in high level/demand enviroments?
Does anyone know a (I don't like the following term, but...) "big player"  who uses it?

And you.. have you been using Cronicle for how long? What can you tell me about it?

Thank you.

**Comment by justinmdudley** [comment]
Hi mcardia,

I am a software engineer at one of the largest biotechnology companies in the world. I utilize Cronicle (in a cluster configuration) in my team as the central production scheduler for all sorts of tasks from data integrations to multiple other systems (internal and external), database processes, security jobs, intense computational tasks, archival jobs, automated reporting, and automated rebuild of dev/test environments. I have added custom plugins to our system to add support for python and ansible tasks. It is an extremely configurable platform. We utilize "three modes" where a job can be scheduled, can be manually triggered in the Cronicle UI, or triggered using the Cronicle API from another application (or a combination of these modes). We also heavily use the chain-reaction functionality where you can chain jobs together. I highly recommend this platform and look forward to the next generation scheduler @jhuckaby and his team are working on!

B/R, Justin.

**Reply by mikeTWC1984** [reply]
I also can recommend this tool. Using it for about 2 years I guess, no Issue whatsoever. Running over 100 jobs a day, not that much, but just to confirm it's pretty stable, even after bunch of customization I made. You can check the very first topic in Discussions, somebody said he is using it to run 25K jobs / 250 events daily.

**Reply by mcfetz** [reply]
That was me. ;-) Still very happy with cronicle. Currently we have 304 active events and ~ 27.000 jobs per day. All is running smoothly without any problems. I can recommend cronicle without hesitation.

**Comment by jhuckaby** [comment]
Thank you all for the endorsements!  That is so awesome.

Just adding one other, from a PR to add a "Companies Using Cronicle" section to the docs: https://github.com/jhuckaby/Cronicle/pull/212
